Services/AzureManagementAdapter/AzureManagementAdapter.cs

```diff
@@ -87,6 +79,37 @@ public async Task<MetricsResponseListModel> PostAsync(MetricsRequestListModel re
             return JsonConvert.DeserializeObject<MetricsResponseListModel>(response.Content);
         }
 
+        public async Task CreateOrUpdateVmssAutoscaleSettingsAsync(int vmCount)
```

Comment:
If there is similar code that could be de-duplicated with a helper, we should do this now, instead of putting it off until some other time that may or may not come.

Reply:
Existing code (tokens, metrics, mgmt adapter), as mentioned earlier, opened a WI for refactor.